### ISSUE TITLE
ID reseting bug

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5913]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
There's this random bug that many ppl seem to experience with random IDs from time to time. I had many IDs randomly reseting during my time on this server, it happens every month/few weeks with a random ID. I had either ulduar/icc/ruby sanctum/voa ID's reseting, sometimes i did them in the weekend or friday and they reseted during the next week, now for example i did ruby sanctum yesterday (wednesday) and today (thursday) the ID reseted. No idea how to reproduce this since its so random with any ID and usually its just 1 ID that reset per week if it does.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
